extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -538,33 +545,77 @@ export class AmpA4A extends AMP.BaseElement {
   }
 
   /**
-   * Try to validate creative is AMP through crypto signature.
-   * @param {!ArrayBuffer} creative  Bytes of the entire signed creative.
-   * @param {?ArrayBuffer} signature  Bytes for creative signature (decoded from
-   *   base64, if necessary.)
-   * @return {!Promise<ArrayBuffer>}  Promise to a guaranteed-valid AMP creative
-   *   or null if the creative is unsigned or invalid.
+   * To be overridden by network specific implementation indicating which
+   * signing service(s) is to be used.
+   * @return {!Array<string>} A list of signing services.
+   */
+  getSigningServiceNames() {
+    // TODO(levitzky) Add dev key name once it goes live.
+    return getMode().localDev ? ['google'] : ['google'];
```

Comment:
The localDev case will be changed once we have an endpoint to call for the dev key. For now, while we hardcode the dev key, the two cases are identical.